rosdep/python.yaml

```
@@ -6273,6 +6280,13 @@ python3-mock:
   openembedded: [python3-mock@meta-ros-common]
   rhel: ['python%{python3_pkgversion}-mock']
   ubuntu: [python3-mock]
+python3-more-itertools:
+  arch: [python-more-itertools]
+  debian: [python3-more-itertools]
+  fedora: [python3-more-itertools]
+  gentoo: [dev-python/more-itertools]
+  rhel: [python3-more-itertools]
```

based on the others rhel rules, I think this must be:

```suggestion
  rhel: ['python%{python3_pkgversion}-more-itertools']
```

---

The link you included showed `python3-more-itertools`. But happy to stay in sync with the rest!